OutOfSchool/OutOfSchool.BusinessLogic/Services/Database/WorkshopService.cs

```diff
@@ -1133,15 +1133,15 @@ await UpdateStatus(new()
 
     private async Task<Workshop> CheckDtoAndPrepareCreatedWorkshop(WorkshopCreateRequestDto dto)
     {
-        if (dto.MemberOfWorkshopId.HasValue && !await Exists((Guid)dto.MemberOfWorkshopId).ConfigureAwait(false))
+        if (dto.ParentWorkshopId.HasValue && !await Exists((Guid)dto.ParentWorkshopId).ConfigureAwait(false))
         {
-            var errorMessage = $"The main workshop (with id = {dto.MemberOfWorkshopId}) for the workshop being created was not found.";
+            var errorMessage = $"The main workshop (with id = {dto.ParentWorkshopId}) for the workshop being created was not found.";
             throw new InvalidOperationException(errorMessage);
         }
 
-        if (dto.MemberOfWorkshopId.HasValue && (await workshopRepository.GetById((Guid)dto.MemberOfWorkshopId).ConfigureAwait(false)).MemberOfWorkshopId.HasValue)
+        if (dto.ParentWorkshopId.HasValue && (await workshopRepository.GetById((Guid)dto.ParentWorkshopId).ConfigureAwait(false)).ParentWorkshopId.HasValue)
```

Comment:
Call to `Exists()` followed by `GetById()`. Could there be better iplementation from performance perspective?